olp-cpp-sdk-core/src/http/curl/NetworkCurl.h

```diff
@@ -19,6 +19,7 @@
 
 #pragma once
 
+#include <curl/curl.h>
```

Comment:
is this needed?

Reply:
I guess yes, cause we use CURL related types in `NeworkCurl::RequestHandle` struct and in other callbacks.

Reply:
But <curl/curl.h> is part of libcurl and should not be related to RequestId, the only change i see here.

Reply:
Missed that it was just moved. Ignore my comment.